### ISSUE TITLE
[Feature Request] Ability to order SSH Connections

### ISSUE THREAD

**Issue by bkno** [Eugeny/terminus#824]
I have added SSH connections for my various servers, I would like to order them by name or drag-and-drop. E.g. Web 1, Web 2, DB 1, DB 2, Redis 1, Redis 2, Redis 3. Currently the order seems to be the order that I created them which was a bit random.

PS - great job with the work you've done on Terminus so far.

**Comment by vogtmh** [comment]
Yes, would be also nice if I would be able to group the SSH connections! And maybe have the possibility to save that configuration encrypted on a Google Drive or any other storage. Termius (without n) works this way and saved me a lot of time so far :-) 

Right now it's more like a "recent connections" list, I guess. 

Apart from that, Terminus looks really promising! I love the interface and the workflow :-)

**Comment by nebubenceszabo** [comment]
Hy! This feature would be really cool!
Question: Is there any workaround to reorder the SSH connections via modifying files of the app?

**Comment by sburkett** [comment]
> Hy! This feature would be really cool!
> Question: Is there any workaround to reorder the SSH connections via modifying files of the app?

I haven't tried this yet, but you could probably reorder them manually in `%APPDATA%\Roaming\Terminus\config.yml` ... be sure to make a backup copy of it first, though, just in case.
